src/Microsoft.AspNet.Mvc.Abstractions/Formatters/IInputFormatter.cs

```diff
@@ -6,25 +6,26 @@
 namespace Microsoft.AspNet.Mvc.Formatters
 {
     /// <summary>
-    /// Reads an object from the request body.
+    /// Reads an <c>object</c> from the request body.
     /// </summary>
     public interface IInputFormatter
     {
         /// <summary>
-        /// Determines whether this <see cref="IInputFormatter"/> can de-serialize
-        /// an object of the specified type.
+        /// Determines whether this <see cref="IInputFormatter"/> can deserialize an <c>object</c> of the
```

Comment:
I'm not really sure what the right thing notationally is to do here - I just wanted to point out my concern. I trust whatever you think is the right way.